game/color.cc

```diff
@@ -52,24 +52,24 @@ Color::Color(std::string const& str) {
 	ColorNames::Map::const_iterator it = colors.m.find(str);
 	if (it != colors.m.end()) { *this = it->second; return; }
 	std::clog << "color/warning: Unknown color: " << str << " (using magenta to hilight)" << std::endl;
-	*this = Color(1.0, 0.0, 1.0);
+	*this = Color(1.0, 0.0, 1.0, 1.0);
 }
 
 glmath::vec4 Color::linear() const {
 	return a * glmath::vec4(r, g, b, 1.0);
 }
 
 Color MicrophoneColor::get(std::string name) {
-	if (name == "black") return Color(3.0/255.0, 3.0/255.0, 3.0/255.0);
-	else if (name == "gray") return Color(24.0/255.0, 24.0/255.0, 24.0/255.0);
-	else if (name == "white") return Color(1.0, 1.0, 1.0);
-	else if (name == "aqua") return Color(0.0, 1.0, 1.0);
-	else if (name == "purple") return Color(63.0/255.0, 0, 1.0);
-	else if (name == "orange") return Color(1.0, 52.0/255.0, 0.0);
-	else if (name == "fuchsia") return Color(1.0, 0.06, 127/255.0);
-	else if (name == "yellow") return Color(1.0, 1.0, 0.0);
-	else if (name == "green") return Color(0.0, 1.0, 0.0);	
-	else if (name == "red") return Color(1, 0.0, 0.0);
-	else if (name == "blue") return Color(0.0, 43.75/255.0, 1.0);
-	else return Color(0.5, 0.5, 0.5);
+	if (name == "black") return Color(3.0/255.0, 3.0/255.0, 3.0/255.0, 1.0);
+	else if (name == "gray") return Color(24.0/255.0, 24.0/255.0, 24.0/255.0, 1.0);
+	else if (name == "white") return Color(1.0, 1.0, 1.0, 1.0);
+	else if (name == "aqua") return Color(0.0, 1.0, 1.0, 1.0);
```

Comment:
so everything has alpha now? making it consistent with everything else. makes sense.

Reply:
Yeah. Less margin for error IMO.